pkg/build/buildkit/buildx.go

```diff
@@ -103,7 +103,7 @@ func (b *Builder) BuildInvocationImage(ctx context.Context, manifest *manifest.M
 
 	out := ioutil.Discard
 	if b.IsVerbose() || b.Config.IsFeatureEnabled(experimental.FlagStructuredLogs) {
-		ctx, log = log.StartSpan("buildkit", attribute.String("source", "porter.build.buildkit"))
+		ctx, log = log.StartSpan(attribute.String("source", "porter.build.buildkit"))
```

Comment:
This seems like a good spot to use StartSpanWithName because we are creating a span within a function, right?